test/distributed/test_c10d_spawn.py

```diff
@@ -8,10 +8,11 @@
 import torch.distributed as c10d
 import torch.multiprocessing as mp
 import torch.nn as nn
+import torch.distributed.nn
```

Comment:
Do you actually need this import since you access them throw torch below?

Reply:
`torch.distributed.nn` can't be accessed if its not directly imported :(

Reply:
Is that expected? Or something we should fix?

Reply:
I have no idea and left it as it was originally devised. I don't mind fixing it in this PR if you guys think its ok.
Or open a new PR so we can discuss this.
Thanks! will address all the comments during the weekend